purchase_order_partner_ref_mandatory/__manifest__.py

```diff
@@ -7,8 +7,8 @@
     """,
     "author": "Solvos",
     "license": "LGPL-3",
-    "version": "15.0.1.0.0",
-    "category": "Inventory/Purchase",
+    "version": "15.0.1.1.0",
+    "category": "Purchase",
```

Comment:
I'd prefer the original category:
```suggestion
    "category": "Inventory/Purchase",
```
because e.g. purchase standard addon has it too (see https://github.com/odoo/odoo/blob/15.0/addons/purchase/__manifest__.py#L7).